docs/dapps/developer-toolchains/using-foundry-with-the-avalanche-c-chain.md

```diff
@@ -156,21 +174,29 @@ Since we are deploying to Fuji testnet, our `RPC_URL` export should be:
 export RPC_URL=https://api.avax-test.network/ext/bc/C/rpc
 ```
 
+If you would like to verify you contracts during the deployment process (fastest and easiest way),
+get a [Snowtrace API Key](https://docs.snowtrace.io/getting-started/viewing-api-usage-statistics).
+Add this as an environment variable:
+
+```zsh
+export ETHERSCAN_API_KEY=<YOUR-SNOWTRACE-API-KEY>
+```
+
 Once set, you can [deploy your NFT with
 Forge](https://book.getfoundry.sh/reference/forge/forge-create.html) by running
 the command below while adding the values for `_name` and `_symbol`, the
 relevant [constructor
 arguments](https://github.com/ava-labs/avalanche-smart-contract-quickstart/blob/3ad93abf50fba65e3aab68f23382bcace73968be/contracts/NFT.sol#L13)
-of the NFT contract:
+of the NFT contract. You can verify the contracts with Snowtrace by adding `--verify` before the `--constructor-args`:
```

Comment:
Does order actually matter? I'm pretty sure you can add `--verify` after the `--constructor-args GateItem ITM` and it will also work.

Reply:
The forge docs recommend `--constructor-args` at the end since it can take multiple parameters
https://book.getfoundry.sh/reference/forge/forge-create